src/lib/setup/prompt.ts

```diff
@@ -237,3 +263,23 @@ export const getAnswerFromFile = (file: string): RequestContext => {
 
   return rc;
 };
+
+export const promptForApprovingHLDPullRequest = async (
+  rc: RequestContext
+): Promise<boolean> => {
+  const urlPR = `${getAzureRepoUrl(
+    rc.orgName,
+    rc.projectName,
+    HLD_REPO
+  )}/pullrequest`;
+  const questions = [
+    {
+      default: true,
+      message: `Please approve and merge a PR at ${urlPR}? Refresh the page after a while if you do not see active PR.`,
```

Comment:
"Please approve and merge the Pull Request at ${urlPR}? Refresh the page if you do not see an active Pull Request."

Reply:
accept